shared/src/business/useCases/processStreamUtilities.js

```diff
@@ -107,7 +107,7 @@ const processCaseEntries = async ({
           },
         },
         NewImage: {
-          ...omit(marshalledCase, 'docketEntries'), // we don't need to store the docket entry list onto the case docket entry mapping
+          ...marshalledCase,
```

Comment:
interesting. just curious about this change. was there a specific reason for it?

Reply:
devex task, the `marshalledCase` is no longer the full case with all of the docketEntries embedded in it.  There is a refactoring above which only fetches the case metadata, the privatePractitioners, and the irsPractitioners since those are the only case attributes we also index into elasticsearch.